services/petstore/pkg/resource/pet/descriptor.go

```diff
@@ -0,0 +1,166 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package pet
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sapirt "k8s.io/apimachinery/pkg/runtime"
+	k8sctrlutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	acktypes "github.com/aws/aws-service-operator-k8s/pkg/types"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+
+	svcapitypes "github.com/aws/aws-service-operator-k8s/services/petstore/apis/v1alpha1"
```

Comment:
nit: we follow practice to separate imports by standard, internal and external.

I got a very good suggestion from Amine earlier. Following link will provide more details. basically using goimports or go fmt helps format imports.
https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Vijay just copied the existing bookstore code :)

Reply:
Thanks! Updating in next pull request since this pull request is merged now.